python/tvm/relay/op/nn/_nn.py

```diff
@@ -205,10 +205,10 @@ def alter_op_layout_conv2d(attrs, inputs, tinfos):
     return topi.nn.conv2d_alter_layout(attrs, inputs, tinfos, op)
 
 @reg.register_legalize("nn.conv2d")
-def legalize_conv2d(attrs, inputs, arg_dtypes):
+def legalize_conv2d(attrs, inputs, types):
     """Legalize conv2d"""
```

Comment:
can you also add the doc string here.

Reply:
Good idea. Let me do it.

Reply:
I understand that this method inherits params from `def conv2d_legalize(attrs, inputs, types, F):`. for convenience, we can add that the doc string comes from `def conv2d_legalize(attrs, inputs, types, F)`